packages/tree-select/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "@automattic/tree-select",
+	"version": "0.1.0",
+	"description": "A selector library for Redux",
+	"keywords": [
+		"redux",
+		"selector"
+	],
+	"author": "Ben Lowery <ben.lowery@automattic.com>",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-only",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"scripts": {
+		"prepare": "npm run build",
+		"build": "node ../../bin/build-package.js",
+		"clean": "node ../../bin/clean-package.js"
```

Comment:
What are the pros and cons of running `lerna run build` versus a `bin/build-packages.js` script that we had before?

Jest, Babel and Gutenberg use a global script that builds all the subpackages.

With this implementation, the published package will still have these scripts defined in its `package.json`, but they don't make any sense now. The file paths point to `node_modules/bin/build-package.js` when installed into a different project.

Reply:
> What are the pros and cons of running lerna run build versus a bin/build-packages.js script that we had before?

I felt like using `lerna run` was more idiomatic lerna, if there is such a thing. The central script does feel more straight-forward. Using lerna and setting up all these scripts is certainly more to maintain..

> Jest, Babel and Gutenberg use a global script that builds all the subpackages

Yup, though I wonder if they're "good" examples of lerna usage? They barely seem to use it. 

> With this implementation, the published package will still have these scripts defined in its package.json, but they don't make any sense now. The file paths point to node_modules/bin/build-package.js when installed into a different project.

If that's an issue, we could move the build scripts to a package, depend on that package, and use it via `npm run`. 

I'm not sure it's really an issue though?

Reply:
> I'm not sure it's really an issue though?

Probably not. I'm just thinking aloud and wondering what's the "right" and "idiomatic" use of Lerna. I'm afraid there is no such thing. Lerna seems to me like a random bag of multirepo utilities rather than some framework-ish system. Not neccessarily a bad thing.

Reply:
>  I'm just thinking aloud and wondering what's the "right" and "idiomatic" use of Lerna. I'm afraid there is no such thing.

That's the same conclusion I'm coming to. Having a solid workflow defined, regardless of how lerna fits in, seems most important.